test/ui-test/testSuites/suite_wallet/shared/steps/walletSteps.py

```diff
@@ -15,36 +17,110 @@
 @Given("the user accepts the signing phrase")
 def step(context):
     the_user_accepts_the_signing_phrase()
-    
-@Given("the user clicks on the first account")
-def step(context):
-    _walletScreen.click_first_account()
-    
-@Given("the user opens wallet screen")
-def step(context):
-    wallet_init_steps.the_user_opens_wallet_screen()
 
 #########################
 ### ACTIONS region:
 #########################
 
-@When("the user adds watch only account \"|any|\" named \"|any|\" and authenticated using password \"|any|\"")
-@verify_screenshot
-def step(context, address, account_name, root_password):
-    _walletScreen.add_watch_only_account(account_name, address, root_password)
-
-@When("an account named \"|any|\" is generated and authenticated using password \"|any|\"")
-def step(context, account_name, password):
-    _walletScreen.generate_new_account(account_name, password)
-
-@When("an account named \"|any|\" is added via private key \"|any|\" and authenticated using password \"|any|\"")
-def step(context, account_name, private_key, password):
-    _walletScreen.import_private_key(account_name, password, private_key)
+@When("the user opens wallet section")
+def step(context):
+    wallet_init_steps.the_user_opens_wallet_screen()
     
-@When("an account named \"|any|\" is added via imported seed phrase \"|any|\" and authenticated using password \"|any|\"")
-def step(context, account_name, mnemonic, password):
-    _walletScreen.import_seed_phrase(account_name, password, mnemonic)
+@When("the user clicks on the default wallet account")
+def step(context):
+    _walletScreen.click_default_wallet_account()
 
+@When("the user adds a watch only account \"|any|\" with \"|any|\" color \"|any|\" and emoji \"|any|\" via \"|any|\"")
+@verify_screenshot
+def step(context, address, name, color, emoji, via_right_click_menu):
+    if via_right_click_menu == VALUE_YES:
+        _walletScreen.click_option_from_left_part_right_click_menu(MainWalletRightClickMenu.ADD_WATCH_ONLY_ACCOUNT_ACTION_PLACEHOLDER.value)
+    else:
+        _walletScreen.open_add_account_popup()
+    _walletScreen.add_account_popup_change_account_name(name)
```

Comment:
Just minor suggestion. Up to you!
```suggestion
    _walletScreen.add_account_popup_change_account_settings(name, color,  emoji)
```

Reply:
I deliberately wanted to have separate function for each field which may be changed from the add/edit account popup, which gives us a possibility to write more granular tests.